inspire/modules/authors/recordext/fields/author.cfg

```diff
@@ -0,0 +1,412 @@
+        {'_curators_note': {'type': 'string'}}
+    creator:
+        marc, "667__", value['a']
+    producer:
+        json_for_marc(), {"667__", ""}
+
+experiments:
+    """Contains information about experiments.
+
+    Currently the ``id`` subfield stores the name of the experiment. It should
+    be moved to storing id of the experiment as soon as experiments are
+    available as records.
+    """
+    schema:
+        {'experiments': {'type': 'list', 'force': True, 'items': {
+         'id': {'type': 'string'},
+         'start_year': {'type': 'integer', 'nullable': True,
+                        'min': __import__('flask').current_app.config['INSPIRE_YEAR_MIN'],
+                        'max': __import__('flask').current_app.config['INSPIRE_YEAR_MAX']},
+         'end_year': {'type': 'integer', 'nullable': True,
+                      'min': __import__('flask').current_app.config['INSPIRE_YEAR_MIN'],
+                      'max': __import__('flask').current_app.config['INSPIRE_YEAR_MAX']},
+         'status': {'type': 'string', 'nullable': True,
+                    'allowed': __import__('flask').current_app.config['INSPIRE_EXPERIMENT_STATUS']}
+         }}}
+    creator:
+        marc, "693__", {'id': value['e'], 'start_year': to_int(value['s']),
+                        'end_year': to_int(value['d']), 'status': value['z']}
+    producer:
+        json_for_marc(), {"693__e": "id", "693__s": "start_year",
+                          "693__d": "end_year", "693__z": "status"}
+
+
+phd_advisors:
+    """Contains a links to the advisors in form of record ids.
+
+    TO DO: define a type for the elements (int). Add creator and producer.
+    """
+    schema:
+        {'phd_advisors': {'type': 'list', 'force': True}}
+
+conferences:
+    """Contains information about conferentions attended.
+
+    TO DO: creator and producer. Move to records.
+    """
+    schema:
+        {'conferences': {'type': 'list', 'force': True}}
+
+urls:
+    """Contains a link to author's website.
+
+    In JSON stored as a list of strings.
+
+    TO DO: define a type and add regex for the elements of the array.
+    """
+    schema:
+        {'urls': {'type': 'list', 'force': True}}
+    creator:
+        marc, "8564_", value['u']
+    producer:
+        json_for_marc(), {"8564_": ""}
+
+date_added:
+    """The date when the record was submitted in form yyyy-mm-dd."""
+    schema:
+        {'date_added': {'type': 'string',
+                        'regex': __import__('flask').current_app.config['INSPIRE_DATE_REGEX']}}
+    creator:
+        marc, "961__", value['x']
+    producer:
+        json_for_marc(), {"961__x": ""}
+
+date_updated:
+    """The latest date when the record was updated in form yyyy-mm-dd."""
+    schema:
+        {'date_updated': {'type': 'string',
+                          'regex': __import__('flask').current_app.config['INSPIRE_DATE_REGEX']}}
+    creator:
+        marc, "961__", value['c']
+    producer:
+        json_for_marc(), {"961__c": ""}
```

Comment:
> If you want to remove it, discuss firstly with Anette.

I dont want to remove it, it is just that I dont see the difference with the one in the base record. Cant we just use one of them?
(and/or override it, so that is populated with the 961 field?)